irods/helpers/__init__.py

```diff
@@ -0,0 +1,50 @@
+import functools
+import inspect
+import types
+import contextlib
+from ..test.helpers import make_session as make_test_session
+from irods.message import (ET, XML_Parser_Type)
+
+
+# See https://stackoverflow.com/questions/67028108/overriding-function-signature-in-help-when-using-functools-wraps
+def update_func(f, func_name='', update_kwargs: dict = None):
+    """Based on http://stackoverflow.com/a/6528148/190597 (Glenn Maynard)"""
```

Comment:
The name of the person who presented that solution is attached to the stackoverflow post. Let's remove their name from the PRC source.

Reply:
removed